rules/corp/rules.go

```diff
@@ -12,6 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// Package corp provides implementations of corp-specific proto rules.
```

Comment:
Changed it to `Package corp contains a list of rules that apply to Google Corp Eng APIs.`